numpy/random/meson.build

```diff
@@ -52,6 +52,11 @@ if host_machine.system() == 'cygwin'
   c_args_random += ['-Wl,--export-all-symbols']
 endif
 
+cython_args = []
+if cy.version().version_compare('>=3.1.0')
```

Comment:
You use also `a0` in some places.  I guess both work, but if you touch it might be nice to align.

Reply:
I use `a0` in the setup.py file because the  PEP440 version parser respects that. Meson versions don't contain the `a0` bit, so it's incorrect to include it here.